tests/gates_cirq_testfixture.h

```diff
@@ -27,7 +27,7 @@ namespace qsim {
 namespace CirqCircuit1 {
 
 template <typename fp_type>
-Circuit<Cirq::GateCirq<fp_type>> GetCircuit(bool qsimh) {
+Circuit<Cirq::GateCirq<fp_type>> GetCircuit(bool qsim) {
```

Comment:
Was the parameter name here previously incorrect?

Reply:
Thank you for the review. Yes, the parameter name here was incorrect.